tests/parsing/grammar/expressions/jsx.js

```diff
@@ -20,6 +20,7 @@ let _ = <div className="menu" onClick={_ => Js.log("click")}></div>
 
 let _ = <Navbar> </Navbar>
 let _ = <Nav.Navbar> </Nav.Navbar>
+let _ = <Nav.navbar> </Nav.navbar>
```

Comment:
Can you add more test cases? Nested, self closing etc.
Can you also test cases for the printer in `tests/printer/expr/jsx.js`?
Can you also add a test case in `tests/parsing/errors/expression/jsx` with an error in the closing "name"?
`<Nav.navbar> </Nav.foo>`?

Reply:
Thanks @IwanKaramazow, added in 6987427.